firestore/google/cloud/firestore_v1/client.py

```diff
@@ -179,6 +191,10 @@ def _rpc_metadata(self):
                 self._database_string
             )
 
+            if self._emulator_host is not None:
+                # The emulator requires additional metadata to be set.
+                self._rpc_metadata_internal.append(("authorization", "bearer owner"))
```

Comment:
I believe this should be `"Bearer owner"`, only the metadata _keys_ need to be lowercase

Reply:
Thanks. It doesn't seem to care one way or the other but I will capitalize it.